packages/design-system-docs/src/pages/components/Pagination/_Pagination.docs.scss

```diff
@@ -0,0 +1,83 @@
+- If the length of the entire collection is less than 3-4 screen lengths long, consider showing all the items at once instead of paginating.
+
+### Usage
+
+- Users want to know the length of a paginated section, so show the size of the paginated set.
+- Highlight the current page the user is on in relation to the entire collection of pages.
+- Don't split the navigation items over multiple lines as this can make individual pages more difficult to understand and select.
+- Don't include out-of-sequence items directly adjacent to one another. Wherever there are missing pages, an ellipses should be used.
+- Avoid adding complexity by focusing on the essentials and avoid adding more items to Pagination just to fill the space.
+- Use touch targets that are big enough to select with any finger and have enough separation to avoid mistakes.
+- Do not use buttons for URL-based pagination. Because the URL is updated, the browser history stack updates and a link is the correct tag to use.
+- Consider page load, performance, and the user's scrolling preferences when determining how many items are displayed on each page.
+
+### Style customization
+
+The following Sass variables can be overridden to theme alerts:
+
+- `$pagination-nav-link-color` - Used to set color of next/previous and page number links.
+- `$pagination-nav-link-color-hover` - Used to set color of next/previous and page number links on hover.
+- `$pagination-nav-link-color-active` - Used to set color of next/previous and page number links when active.
+- `$pagination-nav-link-color-focus` - Used to set color of next/previous and page number links on focus.
+
+- `$pagination-current-page-color` - Used to set the current page element text color.
+- `$pagination-current-page-color-bg` - Used to set the current page element baclground color.
+- `$pagination-overflow-color` - Used to set the overflow element text color.
+- `$pagination-page-count-color` - Used to set the page count element text color.
+
+### Accessibility
+
+- Use a wrapping `<nav>` element to identify Pagination as a navigation section.
+- Pagination **must** have an ARIA label attribute defined on the `<nav>` element that describes its purpose.
+- Use an unordered list for the navigation items as this allows screen readers to voice the number of elements in the Pagination component.
+- Use `aria-current="page"` on the current page's element to properly voice the current page for screen readers.
+- Voice the word "page" before the page numbers.
+- If *Previous* navigation link has keyboard focus, pressing <kbd>TAB</kbd> key **must** move focus to the first pagination page link.
+- If pagination page has keyboard focus, pressing <kbd>TAB</kbd> key **must** move focus to the next pagination page link.
+- If last pagination page has keyboard focus, pressing <kbd>TAB</kbd> key **must** move focus to the *Next* pagination navigation link.
```

Comment:
Does this refer to the pages that are links or the "Next" link at the end?

Reply:
The last page in the visible pagination set. So if Pagination has 20 pages and 12 is active (accounting for ellipses, it'd be 1 ... 11 **[12]** 13 ... 20), if you're tabbing through this set, you'd need to tab from 20 to Next to see the next set of pages (1 ... 12 **[13]** 14 ... 20) - the idea being users should be able to tab from the last page to Next. Not sure if I wrote it to be that clear though

Reply:
Oh, I see now. That wasn't clear to me on my first pass as my brain read "Next" as just next, and not the "Next" link. Could be just me though! If no one else is confused, we're good.